librz/core/cbin.c

```diff
@@ -2946,6 +2946,36 @@ RZ_API bool rz_core_bin_whole_strings_print(RZ_NONNULL RzCore *core, RZ_NONNULL
 	return res;
 }
 
+RZ_IPI bool rz_core_bin_xrefs_strings_print(RZ_NONNULL RzCore *core, RZ_NONNULL RzBinFile *bf, RZ_NONNULL RzCmdStateOutput *state) {
+	rz_return_val_if_fail(core && state, false);
+
+	RzPVector *l = rz_core_bin_whole_strings(core, bf);
+	RzPVector *ll = rz_pvector_new(NULL);
```

Comment:
Please rename `ll` and `l` to something more descriptive. It is important to get into the habit of naming variables properly.

Reply:
Also, the new pvector needs the free function for `RzBinString` passed. Otherwise we get leaks.

Reply:
You can check for leaks by building with ASAN or using valgrind.